src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -63,7 +63,16 @@ public WidgetIndentsHighlightingPassFactory(@NotNull Project project) {
     this.project = project;
     flutterDartAnalysisService = FlutterDartAnalysisServer.getInstance(project);
     this.editorOutlineService = ActiveEditorsOutlineService.getInstance(project);
-    this.inspectorGroupManagerService = InspectorGroupManagerService.getInstance(project);
+    if (!project.getClass().getSimpleName().startsWith("Embed")) {
```

Comment:
It would be nice to have a cleaner solution for this.
Can't we instead make InspectorGrpuManagerService.getInstance(project) more robust so we don't have to skip it?